### ISSUE TITLE
CKEditor wrong input encoding 

### ISSUE THREAD

**Issue by aureliaalessio** [primefaces-extensions/primefaces-extensions.github.com#756]
Hello. I would like to report a **bug**.

When I used to use `p:editor`, I got the HTML by JavaScript function `saveHTML `and it worked perfectly even when the text contained Czech characters (ěščřžýáíéó), I did not even have to set enconding or anything else and it worked. Now however when a user writes "_V případě dalších dotazů se na nás můžete obracet každý den na telefonním čísle_", the gotten HTML from `pe:ckEditor ` has the text like this: "_V pÅípadÄ dalších dotazÅ¯ se na nás mÅ¯Å¾ete obracet kaÅ¾dý den na telefonním Äísle_" - complete rubbish that cannot be further used.

I tried to set the encoding right (though documentation of the CKEditor 4 says that UTF-8 is used by default) in custom config js file, but that did not help. I also tried, after being given the suggestion at [StackOverflow](https://stackoverflow.com/questions/59608699/primefaces-extensions-ckeditor-attempts-to-set-encoding-to-utf-8-unsuccessful), to use [`CharacterEncodingFilter `](http://showcase.omnifaces.org/filters/CharacterEncodingFilter) from OmniFaces which is supposed to be able to fix it, but that did not help either.

The full research and history of me trying to figure out how to fix it is available here: [https://stackoverflow.com/questions/59608699/primefaces-extensions-ckeditor-attempts-to-set-encoding-to-utf-8-unsuccessful](https://stackoverflow.com/questions/59608699/primefaces-extensions-ckeditor-attempts-to-set-encoding-to-utf-8-unsuccessful).

Please let me know if there is any workaround or how can I set the encoding right or if you can fix it. 

Thank you.

PrimeFaces Extensions - version 7.0.2; PrimeFaces - version 7.0.7

**Comment by melloware** [comment]
@aureliaalessio Can you download this ZIP.

[pfe-756.zip](https://github.com/primefaces-extensions/primefaces-extensions.github.com/files/4035323/pfe-756.zip)

Explode it and run "mvn clean jetty:run-exploded" and navigate to http://localhost:8080/primefaces-test/test.xhtml

That is a working mini test project.  Can you then edit it to show me the exact issue you are having and re-attach the ZIP file here after you have updated my sample?

**Comment by aureliaalessio** [comment]
I'll try it right away, @melloware.

**Comment by aureliaalessio** [comment]
So, I and my colleague **found out what the issue was**. The original editor `p:editor`, which we had been using and which we are trying to replace by `pe:ckEditor`, could provide us with its content in HTML only in case we used JavaScript function `saveHTML`. But with `pe:ckEditor`, anytime the user hit the _Send_ button, whose `onstart `contained the `saveHTML`, **the `saveHTML` corrupted the content. Once we erased the `saveHTML` and took the `pe:ckEditor `content as it was (which is already in HTML), it was fine without corrupted characters.**

Thank you @melloware for your time and effort! 

I believe this issue can be closed.
